packages/box/lib/utils/unbox.ts

```diff
@@ -148,7 +148,7 @@ function installBoxDependencies({ hooks }: boxConfig, destination: string) {
   const postUnpack = hooks["post-unpack"];
 
   if (postUnpack.length === 0) return;
-  execSync(postUnpack, { cwd: destination });
+  execSync(postUnpack, { cwd: destination, stdio: "ignore" });
```

Comment:
Would increasing `maxBuffer` be better? Ignoring stdio altogether hides warnings.

Or use spawn instead of exec.

Reply:
Would prefer using spawn to get around the buffer issue. Thanks, @cliffoo

Reply:
Thanks for the review ! I'll push an update using `spawn` tomorrow 😃

Reply:
Looks like parsing can be avoided by setting `shell` to `true` ([gist](https://gist.github.com/cliffoo/2bd76a69fbff35263e1fd32349d25d83))

Reply:
`execSync` will only output `stderr` to the parent process stderr. Should I replicate this behaviour with `spawn` ?